Live_Tests_16_09/Live_Test_GUI_Automated.py

```
@@ -0,0 +1,272 @@
+            self.gui.current_document = self.test_document
+            self.gui.file_path_var.set(str(self.test_document))
+
+            # Check if file path is set
+            if self.gui.current_document == self.test_document:
+                self.log_test("File Selection", True, f"Document path set: {self.test_document}")
+                return True
+            else:
+                self.log_test("File Selection", False, "Document path not set correctly")
+                return False
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)